inference-engine/src/inference_engine/ie_network_reader.cpp

```diff
@@ -151,6 +152,22 @@ void assertIfIRv7LikeModel(std::istream & modelStream) {
 }  // namespace
 
 CNNNetwork details::ReadNetwork(const std::string& modelPath, const std::string& binPath, const std::vector<IExtensionPtr>& exts) {
+    // Try to load with FrontEndManager
+    ngraph::frontend::FrontEndManager manager;
```

Comment:
@nosovmik, where should we move this code to? You told about some special place.

Reply:
I suggest to discuss it with Ilya Lavrenov (not a member of my fork)

Reply:
I think first of all we need to add new tests for this logic.
Moreover in the line 175 we fix unicode name, I think we also need to fix unicode for frontends.